provd/proto/accessibility.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package accessibility;
+
+service AccessibilityService {
+    rpc GetHighContract(Empty) returns (GetHighConstrastResponse) {}
+    rpc SetHighContract(SetHighConstrastRequest) returns (Empty) {}
+    rpc GetReducedMotion(Empty) returns (GetReducedMotionResponse) {}
+    rpc SetReducedMotion(SetReducedMotionRequest) returns (Empty) {}
+    rpc GetScreenReader(Empty) returns (GetScreenReaderResponse) {}
+    rpc SetScreenReader(SetScreenReaderRequest) returns (Empty) {}
```

Comment:
`SetScreenReader` seems slightly off ... could just be me. We're not setting the screen reader per se, we're actually enabling / disabling it no?

`SetScreenReaderState` is slightly better imo, but you could also explore  `ActivateScreenReader`, `DeactivateScreenReader` and `GetScreenReaderState`.

I think a similar comment applies to the other methods.